GHzDACs/Cleanup/dac.py

```diff
@@ -125,70 +183,84 @@ def connect(self, name, group, de, port, board, build):
         p.timeout(self.timeout)
         p.listen()
         yield p.send()
-        
+
         # Get board specific information about this device.
         # We talk to the labrad system using a new context and close it when
         # done.
         reg = self.cxn.registry
         ctxt = reg.context()
         p = reg.packet()
-        p.cd(['','Servers','GHz FPGAs'])
-        p.get('dac'+self.devName.split(' ')[-1], key='boardParams')
+        p.cd(['', 'Servers', 'GHz FPGAs'])
+        p.get('dac' + self.devName.split(' ')[-1], key='boardParams')
         try:
             resp = yield p.send()
             boardParams = resp['boardParams']
             self.parseBoardParameters(boardParams)
         finally:
             yield self.cxn.manager.expire_context(reg.ID, context=ctxt)
-    
+
     @inlineCallbacks
     def shutdown(self):
         """Called when this device is to be shutdown."""
         yield self.cxn.manager.expire_context(self.server.ID,
                                               context=self.ctx)
-    
+
+    def runSram(self, dataIn, loop, blockDelay):
+        raise NotImplementedError()
+
     # Register byte methods
-    
+
     @classmethod
     def regPing(cls):
         """Returns a numpy array of register bytes to ping DAC register"""
         regs = np.zeros(cls.REG_PACKET_LEN, dtype='<u1')
-        regs[0] = 0 # No sequence start
-        regs[1] = 1 # Readback after 2us
-        return regs 
-    
+        regs[0] = 0  # No sequence start
+        regs[1] = 1  # Readback after 2us
+        regs[51] = MONITOR_0
+        regs[52] = MONITOR_1
+        return regs
+
     @classmethod
     def regPllQuery(cls):
         """Returns a numpy array of register bytes to query PLL status"""
         regs = np.zeros(cls.REG_PACKET_LEN, dtype='<u1')
-        regs[0] = 1 # No sequence start
-        regs[1] = 1 # Readback after 2us
+        regs[0] = 0  # No sequence start
+        regs[1] = 1  # Readback after 2us
+        regs[51] = MONITOR_0
+        regs[52] = MONITOR_1
         return regs
-    
+
     @classmethod
     def regSerial(cls, op, data):
         regs = np.zeros(cls.REG_PACKET_LEN, dtype='<u1')
-        regs[0] = 0 #Start mode = no start
-        regs[1] = 1 #Readback = readback after 2us to allow for serial
-        regs[47] = op #Set serial operation mode to op
-        regs[48:51] = littleEndian(data, 3) #Serial data
+        regs[0] = 0  # Start mode = no start
+        regs[1] = 1  # Readback = readback after 2us to allow for serial
+        regs[47] = op  # Set serial operation mode to op
+        regs[48:51] = littleEndian(data, 3)  # Serial data
+        regs[51] = MONITOR_0
+        regs[52] = MONITOR_1
```

Comment:
Is this fixing previously broken code? Why are these monitor values being put in the superclass?